### DISCUSSION TITLE
need an advise

### DISCUSSION THREAD

**Discussion by Kirtishukla2004** [community/community#33983] [Other Feature Feedback, Questions, & Ideas]
namestey everyone i am from india and 
i need some tips to enhance my problem solving skill
thank you

**Comment by james-villanueva-freee** [comment]
Hi. For me, the best way to improve it is to practice. There are tons of platforms out there where you can practice your problem solving skills in any programming language of your chosing. One of my favorites is [codewars](https://www.codewars.com/). Feel free to give it a try 🙇🏼‍♂️
